ding/model/template/vac.py

```diff
@@ -264,13 +268,17 @@ def compute_actor(self, x: torch.Tensor) -> Dict:
             >>> actor_outputs = model(inputs,'compute_actor')
             >>> assert actor_outputs['logit'].shape == torch.Size([4, 64])
         """
-        if self.share_encoder:
-            x = self.encoder(x)
+        if isinstance(x, dict):
+            action_mask = x.get('action_mask', None)
```

Comment:
don't use xxx.get('xxx', None), it will hidden some bugs when your input is incorrect

`x['action_mask']`